adafruit_tfmini.py

```diff
@@ -55,7 +61,7 @@ def __init__(self, uart, *, timeout=1):
         self._mode = None
 
     @property
-    def distance(self):
+    def distance(self) -> None:
```

Comment:
Looks like it returns `dist`, which is an `int`

Reply:
looks like it could return nothing and just raise an error as well.....

Reply:
Exceptions don't get included in the type annotation if there is a valid return (even `None`).  If the only thing a function does with 100% certainty is raise an exception (so not even return `None`), we can type annotate it as `NoReturn`.  Doesn't apply here, but seemed relevant to your point.

Reply:
thanks for the clarification, Alec